scripts/fly-authelia/types.ts

```
@@ -0,0 +1,75 @@
+export type CommandRunner = {
+    run: (spec: CommandSpec) => Promise<CommandResult>;
+    runInteractive: (spec: CommandSpec) => Promise<CommandResult>;
+};
+
+export type Prompt = {
+    text: (message: string) => Promise<string>;
+};
+
+export type Fetcher = (
+    input: string,
+    init?: RequestInit
```

ESLint

[error] 'RequestInit' is not defined.

(no-undef)